cmd/object_api_suite_test.go

```diff
@@ -756,7 +755,6 @@ func testGetDirectoryReturnsObjectNotFound(obj ObjectLayer, instanceType string,
 	for i, testCase := range testCases {
 		_, expectedErr := obj.GetObjectInfo(context.Background(), bucketName, testCase.dir)
 		if expectedErr != nil {
```

Comment:
After removing the `errors.Cause(expectedErr)`, we can combine the 2 if clauses as:
`if expectedErr != nil && expectedErr.Error() != testCase.err.Error() {`